aea/helpers/serializers.py

```diff
@@ -16,13 +16,12 @@
 #   limitations under the License.
 #
 # ------------------------------------------------------------------------------
-
 """This module contains Serializers that can be used for custom types."""
 
 import copy
-from typing import Any, Dict
+from typing import Any, Dict, Tuple
 
-from google.protobuf.struct_pb2 import Struct
+from google.protobuf.struct_pb2 import ListValue, Struct
```

Comment:
Changes like these in a big PR are not ideal. We should be careful not to break anything. Can you submit these changes as a standalone PR please?

Reply:
There was a problem with `DictProtobufStructSerializer` (`_patch_dict` didn't cover nested dictionaries and `list` values) which the new Tac changes exposed. 

We had to make these changes to make Tac work and it was difficult to do this fix in isolation. This bug fix only involves this file and its associated test `tests/test_helpers/test_serializers.py` below. 

If it was a more involved bug fix, I agree that we should have done it separately to reduce confusion.

Reply:
Can we add a few more unit tests to make sure this is now well covered?